### PR TITLE
Fix format table in azure-sql database features-comparison.md

### PR DESCRIPTION
**Pull request by fjvela**

### CODE REVIEW

**Comment by Court72** [comment]
@danimir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/database/features-comparison.md [hunk 1]
```diff
@@ -43,8 +43,7 @@ The following table lists the major features of SQL Server and provides informat
...
-#backup) and [Automated backups in Azure SQL Managed Instance](../managed-instance/automated-backups-overview.md?view=azuresql-mi&preserve-view=true).|
+| [BACKUP command](/sql/t-sql/statements/backup-transact-sql) | No, only system-initiated automatic backups, see [Automated backups in Azure SQL Database](automated-backups-overview.md?view=azuresql-db&preserve-view=true) | Yes, user initiated copy-only backups to Azure Blob storage (automatic system backups can't be initiated by user), see [Backup differences](../managed-instance/transact-sql-tsql-differences-sql-server.md?view=azuresql-mi&preserve-view=true#backup) and [Automated backups in Azure SQL Managed Instance](../managed-instance/automated-backups-overview.md?view=azuresql-mi&preserve-view=true).|
```
Comment (rwestMSFT) [inline azure-sql/database/features-comparison.md:46]:
```suggestion
| [BACKUP command](/sql/t-sql/statements/backup-transact-sql) | No, only system-initiated automatic backups, see [Automated backups in Azure SQL Database](automated-backups-overview.md?view=azuresql-db&preserve-view=true) | Yes, user initiated copy-only backups to Azure Blob storage (automatic system backups can't be initiated by user), see [Backup differences](../managed-instance/transact-sql-tsql-differences-sql-server.md?view=azuresql-mi&preserve-view=true#backup) and [Automated backups in Azure SQL Managed Instance](../managed-instance/automated-backups-overview.md?view=azuresql-mi&preserve-view=true). |
```

**Comment by rwestMSFT** [comment]
#sign-off

**Comment by rwestMSFT** [comment]
Thanks for your contribution, @fjvela!

**Pull request merged**